docs/calibration-solidity.md

```diff
@@ -0,0 +1,229 @@
+## Step 2: Deploy IPC Solidity contracts to Calibration
+
+In order to deploy the latest version of the IPC Solidity contracts, you'll need to pull the following repo: 
+```bash
+https://github.com/consensus-shipyard/ipc-solidity-actors
+cd ipc-soldity-actors
+``` 
+Once inside the repo, you'll need to populate the `.env.template` file with the private key of the address you provided with funds in the previous step, and the endpoint of the target network you want to deploy IPC to (in our case Calibration) : 
+```bash
+export PRIVATE_KEY=<your_private_key>
+export RPC_URL=https://api.calibration.node.glif.io/rpc/v1
+```
+> To export your private key from Metamask you can follow [these steps](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key)
+
+In your currently open terminal, you'll need to load these variables into your environment so you can deploy the contracts.
+```bash
+source .env.template
+make deploy-ipc NETWORK=calibrationnet
+```
+If the deployment is successful, you should receive an output similar to this one: 
+```
+$ ./ops/deploy.sh localnet
+[*] Deploying libraries
+[*] Output libraries available in /home/workspace/pl/ipc-solidity-actors/scripts/libraries.out
+[*] Populating deploy-gateway script
+[*] Gateway script in /home/workspace/pl/ipc-solidity-actors/scripts/deploy-gateway.ts
+[*] Gateway deployed: 
+{ Gateway: '0x2bA26fe8EB6b5C8488132900F00d853ca840F2FB' }
+[*] Output gateway address in /home/workspace/pl/ipc-solidity-actors/scripts/gateway.out
+[*] Populating deploy-registry script
+[*] Registry script in /home/workspace/pl/ipc-solidity-actors/scripts/deploy-registry.ts
+No need to generate any newer typings.
+Nothing to compile
+No need to generate any newer typings.
+Deploying contracts with account: 0x6BE1Ccf648c74800380d0520D797a170c808b624 and balance: 2999143347078092235924
+registry contract deployed to: 0x221485C0948dAa7C8dC1bCbcf813684C5EC1ecED
+[*] IPC actors successfully deployed
+```
+Keep the addresses of the gateway and the registry contracts deployed, as you will need them to configure the IPC agent in the next step.
+
+>💡If instead of deploying IPC Solidity in Calibration, you want to test them in Spacenet or a local network, the only thing that you need to do is to configure the `RPC_URL` of your `.env` to point to the corresponding network's RPC endpoint, and `make deploy-ipc NETWORK=localnet`.
+
+## Step 3: Configure IPC Agent.
+
+After deploying the IPC contracts, we come out of the contracts repo (i.e. where we cloned the `ipc-agent` repo). Let's now deploy and configure our IPC agent. 
+- Compile the IPC agent
+```bash
+cd ipc-agent
+make build
+cd ..
+```
+- We then to initialize a config template and configure it to connect to our IPC instance in Calibration.
+```toml
+[server]
+json_rpc_address = "0.0.0.0:3030"
+
+[[subnets]]
+id = "/r314159"
+network_name = "calibration"
+
+[subnets.config]
+accounts = ["0x6BE1Ccf648c74800380d0520D797a170c808b624"]
```

Comment:
Since we are integrating evm key stpre now, accounts will auto use the address of the private key, so even if it is configured, I think it will not take effect. But once the PR for key store is merged, `account` will behave exactly like that in fvm and `private_key` will be deprecated.

Reply:
Let's update the docs as part of that PR if you don't mind. Should we make that PR to target this one?

Reply:
The evm key store PR is in progess, once done we can just update the doc or even in a new PR with only the doc updates, should be easier to review that way.

Reply:
Fixed